hack/update-deps.sh

```diff
@@ -20,7 +20,7 @@ OCP_VERSION="release-4.6"
 # The list of dependencies that we track at HEAD and periodically
 # float forward in this repository.
 FLOATING_DEPS=(
-  "github.com/openshift/api@${OCP_VERSION}"
+  "github.com/openshift/api@release-4.7"
```

Comment:
Should we update `OCP_VERSION="release-4.7"` and use it here like in the other PR? Why is it different here?

Reply:
It's different here because we have no K8s version bump in here yet, so the bump of all of the OCP deps becomes much more complex. Hence I went with a more surgical change to also reduce the potential for screwups throughout.

Reply:
OK. Makes sense.